planner/core/plan_to_pb.go

```diff
@@ -110,6 +110,29 @@ func (p *PhysicalSelection) ToPB(ctx sessionctx.Context, storeType kv.StoreType)
 	return &tipb.Executor{Tp: tipb.ExecType_TypeSelection, Selection: selExec, ExecutorId: &executorID}, nil
 }
 
+// ToPB implements PhysicalPlan ToPB interface.
+func (p *PhysicalProjection) ToPB(ctx sessionctx.Context, storeType kv.StoreType) (*tipb.Executor, error) {
+	sc := ctx.GetSessionVars().StmtCtx
+	client := ctx.GetClient()
+	exprs, err := expression.ExpressionsToPBList(sc, p.Exprs, client)
+	if err != nil {
+		return nil, err
+	}
+	projExec := &tipb.Projection{
+		Exprs: exprs,
+	}
+	executorID := ""
+	if storeType == kv.TiFlash {
```

Comment:
If the store type is not tiflash, it should report an error.

Reply:
updated